assets/blocks/quiz/answer-blocks/index.js

```diff
@@ -58,8 +66,16 @@ const questionTypes = {
 				'Add a right answer to this question.',
 				'sensei-lms'
 			),
+			noRightAnswerWhitespace: __(
+				'The value of the right answer can not be blank space.',
```

Comment:
So "can not" is actually one word - "can not". 🙂  I think the message could also use an "a":

> The value of the right answer cannot be a blank space.